Looks awesome! Just a question, I tried to run this with ~0.12.0, and got the following error:

``` term
➜  stormpath-express-react-example git:(server-improvements) ✗ npm start

> stormpath-express-react-example@1.0.0 start /Users/orhedenr/Projects/stormpath/stormpath-express-react-example
> node server.js

/Users/orhedenr/Projects/stormpath/stormpath-express-react-example/node_modules/ora/index.js:2
const chalk = require('chalk');
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/orhedenr/Projects/stormpath/stormpath-express-react-example/server.js:8:11)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)

npm ERR! Darwin 15.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.0
```

Personally I prefer to use the latest Node version. But are you aware that it requires people to use at least Node >= v4.0.0?

--- a/server.js
+++ b/server.js
@@ -92,6 +92,9 @@ app.get('*', function (req, res) {
 spinner.text = 'Starting Dev Sever on port ' + port,
 spinner.start();
 
+app.on('error', failAndExit);
+app.on('stormpath.error', failAndExit);
+
 app.listen(port, function () {
   spinner.succeed();
   spinner.text = 'Initializing Stormpath';
@@ -102,6 +105,4 @@ app.listen(port, function () {
     // Now bring back error logging.
     app.get('stormpathLogger').transports.console.level = 'error';
   });
-}).on('error', failAndExit);
-
-app.on('stormpath.error', failAndExit);
+});

